app/pb_kits/playbook/pb_title_detail/_title_detail.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= content_tag(:div,
     id: object.id,
     data: object.data,
-    class: object.classname(object.kit_class)) do %>
-  <%= object.title %>
-  <%= object.detail %>
+    class: object.classname) do %>
+  <%= pb_rails("title", props: { text: object.title, size: 4 }) %>
+  <%= pb_rails("body", props: { text: object.detail, color: "light" }) %>
```

Comment:
Same here. This was wrapped in a conditional in the view object before. If we remove it here, are we saying that the `detail` is a required prop?